manager/main.py

```diff
@@ -8,6 +8,9 @@
 import connexion
 from connexion.resolver import RestyResolver
 from flask import abort
+from flask import request
```

Comment:
Use the `connexion.request` object instead of `flask.request`. Connexion is wrapping the flask app